docker/Dockerfile

```diff
@@ -48,22 +48,13 @@ RUN mkdir -p ${PUDL_INPUT} ${PUDL_OUTPUT} ${DAGSTER_HOME} ${PUDL_REPO}
 # Copy dagster configuration file
 COPY docker/dagster.yaml ${DAGSTER_HOME}/dagster.yaml
 
-# Copy conda-lock.yml in so we can build the conda environment and cache that layer in
-# the Docker image before installing PUDL.
-COPY environments/conda-lock.yml ${PUDL_REPO}/environments/conda-lock.yml
-# Create a conda environment based on the specification in the repo
-RUN micromamba create --prefix ${CONDA_PREFIX} --yes --file ${PUDL_REPO}/environments/conda-lock.yml && \
-    micromamba clean -afy
-
-# Copy the rest of the cloned PUDL repo into the image.
+# Copy the entire PUDL repo into the image.
 # This includes the .git directory, so it is a whole repo
-COPY --chown=${MAMBA_USER}:${MAMBA_USER} . ${PUDL_REPO}
+COPY --chown=${PIXI_USER}:${PIXI_USER} . ${PUDL_REPO}
+WORKDIR ${PUDL_REPO}
 
-# TODO(rousik): The following is a workaround for sudden breakage where conda
-# can't find libraries contained within the environment. It's unclear why!
-ENV LD_LIBRARY_PATH=${CONDA_PREFIX}/lib
-RUN ${CONDA_RUN} pip install --no-cache-dir --no-deps --editable ${PUDL_REPO}
+# Install the pixi environment (includes the editable local package)
+RUN pixi install --locked
 
-WORKDIR ${PUDL_REPO}
 # Run the unit tests:
-CMD ["micromamba", "run", "--prefix", "${CONDA_PREFIX}", "--attach", "''", "pytest", "test/unit"]
+CMD ["pixi", "run", "-v", "pytest-unit"]
```

Comment:
I was able to run the container and it ran the tests.

Reply:
Branch build success also indicates this Dockerfile is working!